src/main/java/core/basesyntax/NumberSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberSupplier {
+    public int getRandomNumber() {
```

Comment:
The method getRandomNumber() could be improved by not creating a temporary variable. You can directly return the result of the Random instance's nextInt() method.